src/Command/FixtureCommand.php

```diff
@@ -419,6 +422,31 @@ protected function _generateRecords(TableSchemaInterface $table, int $recordCoun
                         $insert = Text::uuid();
                         break;
                 }
+                if (str_starts_with($fieldInfo['type'], 'enum-')) {
+                    $insert = null;
+                    if ($fieldInfo['default'] || $fieldInfo['null'] === false) {
+                        $dbType = TypeFactory::build($fieldInfo['type']);
+                        if ($dbType instanceof EnumType) {
+                            $class = $dbType->getEnumClassName();
+                            $reflectionEnum = new ReflectionEnum($class);
+                            $backingType = (string)$reflectionEnum->getBackingType();
+
+                            if ($fieldInfo['default'] !== null) {
+                                $insert = $fieldInfo['default'];
+                                if ($backingType === 'int') {
+                                    $insert = (int)$insert;
+                                }
+                            } else {
+                                $cases = $reflectionEnum->getCases();
+                                $firstCase = array_shift($cases);
+                                /** @var \BackedEnum $firstValue */
+                                $firstValue = $firstCase->getValue();
```

Comment:
Could we hit a runtime error here if the enum has no cases? I don't think there is a practical scenario for no cases, but it could happen.

Reply:
Wrapped it.